arches_her/templates/views/components/reports/consultation.htm

```diff
@@ -47,7 +44,627 @@ <h1 class="aher-report-toolbar-title"><span class="aher-report-name">{% trans "C
+                                                <td data-bind="text: attendees"></td>
+                                                <td data-bind="text: relatedCondition"></td>
+                                                <td data-bind="text: note"></td>
+                                                <td data-bind="text: followOnAction"></td>
+                                                <td><a class="aher-table-link" data-bind="text: digitalFile, attr: {href: digitalFileLink}"></a></td>
+                                                <td class="aher-table-control">
+                                                    <div data-bind="if: $parent.cards.communications">
+                                                        <a data-bind="click: function() {$parent.editTile(tileid, $parent.cards.communications)}">
+                                                            <i class="fa fa-mail-reply"></i>
+                                                        </a>
+                                                        <a data-bind="click:  function() {$parent.deleteTile(tileid, $parent.cards.communications)}">
+                                                            <i class="fa fa-trash"></i>
+                                                        </a>
+                                                    </div>
+                                                </td>
+                                            </tr>
+                                        </tbody>
+                                    </table>
+                                </div>
+                            </div>
+                        </div>
+                        <!-- /ko -->
+                    </div>
+                </div>
+            </div>
+            <!-- Site Visit Tab -->
+            <div class="aher-report-page" data-bind="if: activeSection() === 'sitevisits'">
+                <h2 class="aher-report-section-title"><i data-bind="click: function() {toggleVisibility(visible.siteVisits)}, css: {'fa-angle-double-right': visible.siteVisits(), 'fa-angle-double-up': !visible.siteVisits()}" class="fa toggle"></i> {% trans "Site Visits" %}</h2>
+                <a data-bind="{if: cards['site visits'], click: function(){ addNewTile(cards['site visits']) }}" class="aher-report-a">
+                    <i class="fa fa-mail-reply"></i>
+                    <span data-bind="if: cards['site visits'].tiles().length">{% trans "Edit Site Visits" %}</span>
+                    <span data-bind="ifnot: cards['site visits'].tiles().length">{% trans "Add Site Visits" %}</span>
+                </a>
+
+                <!-- Collapsible content -->
+                <div data-bind="visible: visible.siteVisits" class="aher-report-collapsible-container pad-lft">
+                    <!-- ko ifnot: siteVisits() -->
+                    <div class="aher-nodata-note">No site visit for this consultation</div>
```

Comment:
This is never false, and doesn't show up when empty.  You may want to check for .length.  This is happening in many of the tables right now - the tables show up with "No data in table" rather than the desired message.  You may want to test by starting with an empty consultation resource

Reply:
Fixed and a couple of other places with the similar if statements.